cypress/support/step-definitions/flat-table-steps.js

```diff
@@ -12,118 +12,124 @@ const FIFTH_ELEMENT = 4;
 const SIXTH_ELEMENT = 5;
 const SEVENTH_ELEMENT = 6;
 
-Then('FlatTable has sticky row header', () => {
-  const color = 'rgb(204, 214, 218)';
-  const rowHeaderAmount = 8;
-  for (let i = 1; i <= rowHeaderAmount; i++) {
-    flatTableStickyRowHeader(i).should('have.css', 'border-right-color', color)
+Then('FlatTable has sticky row', () => {
+  flatTableBodyRows().each(($el, index, $list) => {
```

Comment:
do we need parameters `index` and `$list` here?

Reply:
I removed `$list`. But we need `index` - I used it to find sticky row by `index`